jme3-lwjgl/src/main/java/com/jme3/audio/lwjgl/LwjglALC.java

```diff
@@ -12,10 +12,36 @@
 
     @Override
     public void createALC() {
-        try {
-            AL.create();
-        } catch (LWJGLException ex) {
-            throw new RuntimeException(ex);
+        int numRetriesRemaining = 4;
+        int retryDelayMsec = 100; // start with an 0.1-second delay
+
+        while (true) {
+            try {
+                AL.create();
+                break;
+
+            } catch (LWJGLException exception1) {
+                if (numRetriesRemaining < 1) {
+                    throw new RuntimeException(exception1);
+                }
+
+                // Retry to mitigate JME Issue 1383.
+                --numRetriesRemaining;
+                System.out.printf("Caught an LWJGLException from AL.create(). "
+                        + "Will retry after %d msec, "
+                        + "with %d more retr%s remaining.%n",
+                        retryDelayMsec,
+                        numRetriesRemaining,
+                        (numRetriesRemaining == 1) ? "y" : "ies");
+
+                try {
+                    Thread.sleep(retryDelayMsec);
+                } catch (InterruptedException exception2) {
+                }
+
+                // Triple the wait time after each failure.
+                retryDelayMsec *= 3;
```

Comment:
Not sure about this. If it fails a couple of times it might cause the app to lock too much and trigger the os to ask the user to kill it.
I think it would be better to always wait for a very short time (10ms or 100 ms?) and keep spamming it until the library is loaded, to get the app going as soon as possible.

Reply:
Or increase it linearly instead of geometrically.

Reply:
The maximum number of retries is 4. If `create()` always fails, then the total sleep time would be 4 seconds (0.1 + 0.3 + 0.9 + 2.7). Could 4 seconds of sleeping (not busy waiting) during initialization really trigger the OS to intervene? I don't see how.

I implemented increasing sleeps because I don't know how much waiting is needed. What if 100 ms isn't enough time?
I limited the number of retries in case it's a situation where `create()` always fails. In which case, I want:

 + the app to terminate (so it's obvious that something has gone wrong) and
 + the software to generate a stack trace.

Reply:
I am not sure, but i think 4 seconds of waiting on the main thread for graphical applications could be problematic.
What is the reason for increasing the sleeping time in the first place?

Reply:
This is during initialization... not during "application running".  And it seems that sometimes OpenAL has to be hit a few times before it will actually start.

Reply:
Does this happen before the lwjgl context is created?

Reply:
No. The GL context is created first.

Reply:
@riccardobl I'm looking for constructive feedback. What would be an acceptable amount of sleep time, in your opinion?

Reply:
I am not sure about the reason for increasing the sleeping time. Since this has to be loaded as fast as possible, i wouldn't wait more than 100ms. That's more than enough to not hog the cpu.

Reply:
I've pushed a commit to remove the increases. I believe that resolves this conversation. Thank you both for your help!